hw03_Ksmun.py

```diff
@@ -0,0 +1,66 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Naod Ksmu
+# Username: Ksmun
+#
+# Assignment:CSC 226 - HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+ #Google Doc Link:https://docs.google.com/document/d/1wlV5ufYtq_pLxl3pC2goho9RFRjxMk1HN4RcPo24w3w/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: I used ChatGPT, an AI by OpenAI, for guidance on structuring my code and debugging.
+# All final coding decisions were my own.
+#  Citation: OpenAI. (2025). ChatGPT [AI model]. https://openai.com
+#
+#
+#################################################################################
+
+
+import turtle
+
+screen=turtle.Screen()
+screen.colormode(255)
+screen.bgcolor((115, 134, 120))
+
+t = turtle.Turtle()
+t.speed(3)
```

Comment:
There should be no code outside of functions. Please move this into `main` at the beginning.